rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/RMDBManager.java

```diff
@@ -458,16 +474,18 @@ public Void doInTransaction(Session session) {
                 throw new RuntimeException("Exception occurred while removing nodes", e);
             }
         } else {
+            logger.info("Request " + RMDBManagerBuffer.class.getSimpleName() + " to remove nodes " +
+                        Arrays.toString(nodes.toArray()) + " in database");
```

Comment:
I am often afraid to print a list of nodes.
Maybe we can print the number of nodes in info, and print the list in debug ?

Reply:
yes I agree it seems better